buildpipeline/security/DotNet-Core-Setup-Security-Windows.json

```diff
@@ -560,11 +560,11 @@
         "imageUrl": "https://devdiv.visualstudio.com/_api/_common/identityImage?id=9d5fdf9f-36b6-4d0c-a12e-2737a673af94"
         },
     "queue": {
-        "id": 36,
-        "name": "DotNet-Build",
+        "id": 330,
```

Comment:
corefx and coreclr don't use this pool.  Why do we?

/cc @mmitche

Reply:
@eerhardt They should be soon.